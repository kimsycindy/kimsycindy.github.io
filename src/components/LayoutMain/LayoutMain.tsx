import React from 'react'
import cx from 'classnames'

import styles from './LayoutMain.module.scss'

interface Props {
  className?: string
}

const LayoutMain: React.FC<Props> = ({ children, className }) => (
  <div className={cx(styles.LayoutMain, className)}>{children}</div>
)

export default LayoutMain
