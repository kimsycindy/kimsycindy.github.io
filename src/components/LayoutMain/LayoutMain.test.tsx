import React from 'react'
import renderer from 'react-test-renderer'

import LayoutMain from './LayoutMain'

describe('LayoutMain', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LayoutMain className="layoutMain-mock">
          <div>LayoutMain</div>
        </LayoutMain>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
