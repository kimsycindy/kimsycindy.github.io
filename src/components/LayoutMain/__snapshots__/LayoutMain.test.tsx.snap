// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMain renders correctly 1`] = `
<div
  className="LayoutMain layoutMain-mock"
>
  <div>
    LayoutMain
  </div>
</div>
`;
